MozillaRustComponentsWrapper/dummy.m

```diff
@@ -0,0 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is there a reason why this file is a `.m` file? Just curious, I know it's a dummy file anyways 😄